Shared/Samples/Set visibility of subtype sublayer/SetVisibilityOfSubtypeSublayerView.Model.swift

```diff
@@ -0,0 +1,168 @@
+            labelDefinition.usesCodedValues = true
+            return labelDefinition
+        }()
+        
+        /// The current scale of the map.
+        var currentScale: Double = .zero
+        
+        /// The map's current scale value in text.
+        @Published var currentScaleText: String = ""
+        
+        /// The subtype sublayer's minimum scale value in text.
+        @Published private(set) var minimumScaleText: String = "Not Set"
+        
+        init() {
+            map.initialViewpoint = .initialViewpoint
+            subtypeFeatureLayer = SubtypeFeatureLayer(featureTable: featureTable)
+            subtypeFeatureLayer.scalesSymbols = false
+        }
+        
+        deinit {
+            ArcGISEnvironment.authenticationManager.arcGISCredentialStore.removeAll()
+        }
+        
+        /// Performs important tasks including adding credentials, loading and adding operational layers.
+        func setup() async throws {
+            try await ArcGISEnvironment.authenticationManager.arcGISCredentialStore.add(.publicSample)
+            try await subtypeFeatureLayer.load()
+            map.addOperationalLayer(subtypeFeatureLayer)
+            guard let subtypeSublayer = subtypeFeatureLayer.sublayer(withSubtypeName: "Street Light") else {
+                throw SetupError.cannotFindSublayer
+            }
+            subtypeSublayer.labelsAreEnabled = true
+            originalRenderer = subtypeSublayer.renderer
+            subtypeSublayer.addLabelDefinition(labelDefinition)
+            self.subtypeSublayer = subtypeSublayer
+        }
+        
+        func toggleSublayer(isVisible: Bool) {
+            subtypeSublayer?.isVisible = isVisible
+        }
+        
+        func toggleRenderer(showsOriginalRenderer: Bool) {
+            if showsOriginalRenderer {
+                subtypeSublayer?.renderer = originalRenderer
+            } else {
+                let symbol = SimpleMarkerSymbol(style: .diamond, color: .systemPink, size: 20)
+                let alternativeRenderer = SimpleRenderer(symbol: symbol)
+                subtypeSublayer?.renderer = alternativeRenderer
+            }
+        }
+        
+        func formatCurrentScaleText() {
+            currentScaleText = "1:\(currentScale.formatted(.decimal))"
+        }
+        
+        func setMinimumScale() {
+            minimumScaleText = currentScaleText
+            subtypeSublayer?.minScale = currentScale
+        }
+    }
+}
+
+extension SetVisibilityOfSubtypeSublayerView.Model {
+    enum SetupError: Error {
+        case cannotFindSublayer
+    }
+}
+
+extension SetVisibilityOfSubtypeSublayerView.Model.SetupError: LocalizedError {
+    var errorDescription: String? {
+        switch self {
+        case .cannotFindSublayer:
+            return NSLocalizedString(
+                "Cannot find subtype sublayer.",
+                comment: "Error thrown when subtype sublayer cannot be found."
+            )
+        }
+    }
+}
```

Comment:
```suggestion
extension SetVisibilityOfSubtypeSublayerView.Model {
    enum SetupError: LocalizedError {
        case cannotFindSublayer
        
        var errorDescription: String? {
            switch self {
            case .cannotFindSublayer:
                return NSLocalizedString(
                    "Cannot find subtype sublayer.",
                    comment: "Error thrown when subtype sublayer cannot be found."
                )
            }
        }
    }
}
```
Should it be simplified like this?